PostfixCalculator/main.c

```diff
@@ -0,0 +1,92 @@
+#include "../Stack/stack.h"
+#include <stdio.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <stdlib.h>
+
+int postfixCalculator(char* expression, int length) {
```

Comment:
```suggestion
int calculatePostfixExpression(const char* expression, int length) {
```
- Функции именуются глаголами в повелительной форме
- const даст возможность использовать строковые литералы, например, `calculatePostfixExpression("1 1 +", 5);`